hmt_escrow/job.py

```diff
@@ -26,6 +27,7 @@
 from hmt_escrow.storage import download, upload, get_public_bucket_url, get_key_from_url
 
 GAS_LIMIT = int(os.getenv("GAS_LIMIT", 4712388))
+BULK_TRANSFER_EVENT = get_entity_topic("HMToken.sol:HMToken", "BulkTransfer")
```

Comment:
Transfer event

Reply:
in erc20 interface file, there's Transfer event

Reply:
HMTokenInterface.sol

Reply:
done